pi-manage

```diff
@@ -663,6 +667,37 @@ def rotate_audit(highwatermark=10000, lowwatermark=5000, age=0, config=None,
             print("{0!s} entries deleted.".format(r))
 
 
+@contextlib.contextmanager
+def smartopen(filename):
+    if filename and filename != '-':
+        fh = open(filename, 'w')
+    else:
+        fh = sys.stdout
+
+    try:
+        yield fh
+    finally:
+        if fh is not sys.stdout:
+            fh.close()
+
+
+@audit_manager.command
+def dump(filename):
```

Comment:
Please add a parameter to only dump the last ``x`` days. Since we want to use this function also for support cases, it makes sense to only receive some days.

You can use the parameter ``timelimit``  in the ``csv_generator``, which takes a ``Timedelta``, I think.

Something like 

~~~~python
@audit_manager.command
def dump(filename, days):
~~~~
could do the trick or

~~~~python
@audit_manager.option('-f', '--fiilename', default='-')
@audit_manager.option('-d', '--days', default='7')
def dump(filename, days):
~~~~